### ISSUE TITLE
100% CPU use

### ISSUE THREAD

**Issue by lukebakken** [ccavanaugh/jgnash#21]
Hi @ccavanaugh -

I have noticed that jGnash climbs to 100% cpu use after using it a few minutes. Typically I'm entering items into a couple accounts, then reconciling them with my bank's online info. Closing jGnash brings CPU use (and temp!) back to normal.

Environment:

* Arch Linux, all the latest `pacman` updates
* jGnash 2.28.3, FX interface
* Java:
    ```
    $ java -version
    openjdk version "1.8.0_121"
    OpenJDK Runtime Environment (build 1.8.0_121-b13)
    OpenJDK 64-Bit Server VM (build 25.121-b13, mixed mode)
    ```

**Comment by ccavanaugh** [comment]
I found and fixed this a couple of days ago.  I goofed and left some debugging code in place to stress test some threading code.  To be specific, it should start loading the system after about 45 seconds after start.

The commit is [https://github.com/ccavanaugh/jgnash/commit/c1c7702157696e8457396d23ffbb1e2052b7f5bd](url)

Nice to see another Arch user.

Thanks,
Craig

**Comment by lukebakken** [comment]
Great! I'll give it a try as soon as I can.

**Comment by ccavanaugh** [comment]
Closed, fixed in release 2.29.0

**Comment by lukebakken** [comment]
Confirmed with `2.29.0`, thanks again.
